mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1466,6 +1466,15 @@ func resourceContainerCluster() *schema.Resource {
 			   Computed: true,
 			},
 
+<% if version == 'beta' -%>
+			"enable_cost_allocation": {
```

Comment:
Haven't looked at the API documentation yet, but it looks like the API defines this as `CostManagementConfig.Enabled`. We have shied away from flattening fields like this recently because if the API adds more fields later, it becomes a breaking change to correct. For an example of a similar field, I think `mesh_certificates.enable_certificates`.

My recommendation is to expand the field to `cost_management_config.enabled`

Reply:
Thanks, wasn't sure which structure to pick since I saw both being used. Should now be fixed.